src/main/java/com/amazonaws/kinesisvideo/internal/producer/jni/NativeKinesisVideoProducerJni.java

```diff
@@ -568,6 +568,25 @@ public void putFragmentMetadata(final long streamHandle, final @Nonnull String m
         putKinesisVideoFragmentMetadata(mClientHandle, streamHandle, metadataName, metadataValue, persistent);
     }
 
+    /**
+     * Put an event metadata.
```

Comment:
You can add each event type at most once to each fragment otherwise you will receive 0x5200008B.
The metadata key cannot start with AWS